react/features/invite/components/info-dialog/PasswordForm.web.js

```diff
@@ -0,0 +1,171 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+import { translate } from '../../../base/i18n';
+import { LOCKED_LOCALLY } from '../../../room-lock';
+
+/**
+ * React {@code Component} for displaying and editing the conference password.
+ *
+ * @extends Component
+ */
+class PasswordForm extends Component {
+    /**
+     * {@code PasswordForm} component's property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * The value for how the conference is locked (or undefined if not
+         * locked) as defined by room-lock constants.
+         */
+        locked: PropTypes.string,
+
+        /**
+         * Callback to invoke when the local participant is submitting a
+         * password set request.
+         */
+        onSubmit: PropTypes.func,
+
+        /**
+         * The current known password for the JitsiConference.
+         */
+        password: PropTypes.string,
+
+        /**
+         * Whether or not to show the password editing field.
+         */
+        showEditing: PropTypes.bool,
```

Comment:
Very minor: Maybe it's just me, but "showEditing" sounds a little bit weird. Is this more about editEnabled or editModeEnabled?